.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,11 @@
 <!--
-Before opening your pull request, have a quick look at our contribution guidelines: https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
+Before opening your pull request, have a quick look at our contribution guidelines:
+https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
+-->
+
+<!--
+Add a preview image of your submission using
```

Comment:
I would:

- Just include it in the same comment.
- Add a colon at the end, like the existing line.
- Either have an extra empty line in both or in neither.

```suggestion

Add a preview image of your submission using:

```